.github/workflows/container-autoupdate.yml

```diff
@@ -0,0 +1,34 @@
+name: Build, test and push container
+on:
+  schedule:
+    - cron: 0 0 * * *
+  workflow_dispatch:
+
+jobs:
+  push-master-container:
+    name: Refresh Rawhide container
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout anaconda repository
+        uses: actions/checkout@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Login to Quay.io
+        uses: docker/login-action@v1
+        with:
+          registry: quay.io
+          username: ${{ secrets.QUAY_USERNAME }}
+          password: ${{ secrets.QUAY_PASSWORD }}
+
+      - name: Build and push container
+        uses: docker/build-push-action@v2
+        with:
+          context: dockerfile/ci-tasks/
+          file: dockerfile/ci-tasks/Dockerfile
+          platforms: linux/amd64
+          push: true
+          tags: |
+            quay.io/rhinstaller/anaconda-ci:latest
+            quay.io/rhinstaller/anaconda-ci:master
```

Comment:
Again, let's not hardcode `:master` here, but use `$(GIT_BRANCH)` like `Makefile` does. That way you can keep the workflow file in sync between branches. Also, I think `:latest` makes relatively little sense here. At least it should be pushed conditionally only for master.

Reply:
Good point!